hvplot/converter.py

```diff
@@ -334,6 +334,24 @@ class HoloViewsConverter:
     -------------
     bgcolor (default=None): str
         Background color of the data area of the plot
+    color: str or list, optional
+        Color(s) to use for the plot. Accepts a single color, a list of colors, or
+        a column name from the data to map colors based on values.
+    c: str or list, optional
+        Alias for `color`. Used for specifying colors in scatter plots and other
+        elements that support color mapping.
+    cmap: str or list, optional
+        The colormap to use for continuous or categorical color mapping.
+        Accepts colormap names, dictionaries for discrete mapping,
+        colormap objects, or custom lists.
+    colormap: str or list, optional
+        Alias for `cmap`. The colormap to apply when using color mapping.
```

Comment:
Which one is the alias of the other? 🙃 And same question as for color/c, is it a strict alias?

Reply:
Yes, it seems colormap the alias for cmap. See:
```python
        # Color
        cmap_kwds = set(cmap_opts).intersection(kwds)
        if len(cmap_kwds) > 1:
            raise TypeError('Specify at most one of `cmap`, `colormap`, or `color_key`.')

        cmap = kwds.pop(cmap_kwds.pop()) if cmap_kwds else None
        color = kwds.pop('color', kwds.pop('c', None))
```
https://github.com/holoviz/hvplot/blob/8b44a69941a6384f5fd8346b97f993348c6138e6/hvplot/converter.py#L1561-#L1566